### ISSUE TITLE
`no-magic-numbers` should ignore various Number methods' params

### ISSUE THREAD

**Issue by mateuszwitkowski** [palantir/tslint#3653]
### Bug Report

Since linter now ingores radix param in parseInt method maybe it should be good to standardize how params are handled in those Number methods:

- toFixed()
- toExponential()
- toPrecision()

#### TypeScript code being linted

```ts
someNumber.toFixed(2);
someNumber.toExponential(4);
someNumber.toPrecision(2);
```

with `tslint.json` configuration:

```json
{
  "no-magic-number": true
}
```

#### Actual behavior
[tslint] 'magic numbers' are not allowed (no-magic-numbers)

#### Expected behavior
No error.

**Comment by mateuszwitkowski** [comment]
If it's OK I'd like to take care of this but I need some hint: how to get the name of called method?

**Comment by aervin** [comment]
@mateuszwitkowski You should paste these snippets
```ts
someNumber.toFixed(2);
someNumber.toExponential(4);
someNumber.toPrecision(2);
```
into [ast-explorer](http://astexplorer.net/) to get a better idea of the AST nodes you'll be parsing.

**Comment by mateuszwitkowski** [comment]
@aervin That's great tool. Thanks!

**Comment by JoshuaKGoldberg** [comment]
This was fixed by #3668 🎉
